src/Disciplina/DB/DSL/Interpret.hs

```diff
@@ -0,0 +1,99 @@
+module Disciplina.DB.DSL.Interpret
+              ( runQuery
+              , RunQuery (..)
```

Comment:
You should import `MonadSearchTxObj` class here. Could be easily added later on demand, though.